acme/localStore_windows.go

```diff
@@ -1,6 +1,21 @@
 package acme
 
+import "os"
+
+// Check file content size
 // Do not check file permissions on Windows right now
```

Comment:
I think that this comment is no more necessary WDYT ?

Reply:
It's still not possible to check file permission with Windows, that's why I kept it.
I just added check on the the file content.